contracts/peripheral/AuraMining.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.11;
+import { AuraMath } from "../utils/AuraMath.sol";
+
+// Forked of https://etherscan.io/address/0x3c75bfe6fbfda3a94e7e7e8c2216afc684de5343#code
+//  - Refactor based on Aura emissions schedule.
+
+interface ICvx {
+    function reductionPerCliff() external view returns (uint256);
+
+    function totalSupply() external view returns (uint256);
+
+    function totalCliffs() external view returns (uint256);
+
+    function INIT_MINT_AMOUNT() external view returns (uint256);
+
+    function EMISSIONS_MAX_SUPPLY() external view returns (uint256);
+}
+
+/**
+ * @notice Utility library to calculate how many Cvx will be minted based on the amount of Crv
+ */
+library AuraMining {
+    ICvx public constant cvx = ICvx(0xC0c293ce456fF0ED870ADd98a0828Dd4d2903DBF);
+    using AuraMath for uint256;
+
+    /**
+     * @dev Calculates the amount of AURA to mint based on the BAL supply schedule.
+     */
+    function convertCrvToCvx(uint256 _amount) external view returns (uint256 amount) {
+        uint256 supply = cvx.totalSupply();
+        uint256 totalCliffs = cvx.totalCliffs();
+        uint256 maxSupply = cvx.EMISSIONS_MAX_SUPPLY();
+        uint256 initMintAmount = cvx.INIT_MINT_AMOUNT();
+
+        uint256 emissionsMinted = supply - initMintAmount;
```

Comment:
Wonder if we should add a check to see if the `inflationProtectionTime` has passed as this function will only be correct if minterMinted is still 0. At the very least we should leave a comment here to explain.